hack/lib/build/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This library holds utility functions for building releases.
+
+# os::build::release::check_for_rpms checks that an RPM release has been built
+function os::build::release::check_for_rpms() {
+	if [[ ! -d "${OS_LOCAL_RPMPATH}" || ! -s "${OS_LOCAL_RELEASEPATH}/CHECKSUM" ]]; then
```

Comment:
>  and the images have no use for the release binaries anyway.

Or at least they don't have any use since now we switch to placing the RPMs inside the images (I think only the origin-pod and dockerregistry images required the existence of a binary prior to this PR).